image/image.go

```diff
@@ -48,7 +48,12 @@ func (i *genericImage) IntendedDockerReference() string {
 // NOTE: It is essential for signature verification that Manifest returns the manifest from which LayerDigests is computed.
 func (i *genericImage) Manifest() ([]byte, error) {
 	if i.cachedManifest == nil {
-		m, _, err := i.src.GetManifest([]string{manifest.DockerV2Schema1MIMEType})
+		mts := []string{
```

Comment:
We have enough electrons to afford a readable name like `manifestMIMETypes` ☺ or perhaps just have a multi-line argument to `GetManifest`.